### ISSUE TITLE
Phase 5: Visualization: What questions are we trying to answer?

### ISSUE THREAD

**Issue by karanjeets** [polar-computing/AerosolDelta#6]
1. Demonstrate the presence of aerosols on Land Ice?
   1. On a world map, show labels/colors/bubbles to indicate Aerosol presence in a geographical area.
   2. The Map in Step 1 can be attached to a drop down which has a list of Month-Year values. On selecting a month-year, the map will show respective aerosol presence.
   3. Use light, normal, dark color to identify the level of aerosol in each region.

**Please add more questions and related visualizations**

**Comment by karanjeets** [comment]
@kacasey Inputs, please.

**Comment by kacasey** [comment]
Yes, this sounds like a great plan.     If possible, I would also like to output the results in an animation - that moves forward monthly from 2000-present.   And output the results in graphical form, by RGI region, month from Jan 2000 to present, and mean aerosol concentration by type, mean land ice surface reflectance.    (Please see more details added below).

**Comment by r4space** [comment]
Perhaps too much for the hack but maybe one possibility for the visualisation
[NASA Wed World Wind](https://webworldwind.org/get-started/)

Something like [this](http://worldwindserver.net/webworldwind/examples/BlueMarbleTimeSeries.html)?

**Comment by kacasey** [comment]
Sure!  That would be great!   I like how that visualization gives the option to change map projections, and zoom on different areas.   A visualization along those lines with aerosols visualized similar to [here](https://svs.gsfc.nasa.gov/11723), and perhaps a side by side world map showing land ice areas highlighted in gray scale where bright white would indicate high reflectance values, and darker grays lower reflectance values (see example brainstorm sketch image).    Or other ideas welcome for how to visualize ice reflectivity under / in conjunction with surface mass deposition of aerosols.  
![animation-ideas](https://cloud.githubusercontent.com/assets/18668993/16665999/ba3d41d8-4453-11e6-9d2c-8718147cda73.png)

Also, I'd like a graphical representation of surface reflectance over time, by region, and aerosol type over time by region.    Something similar graphically to the attached plot (different variable - plotting glacier mass loss by region). 
![orlove figure 1_small](https://cloud.githubusercontent.com/assets/18668993/16627616/ff0b69ee-437b-11e6-86f4-572957740e9b.jpg)

And / or a world map with annual aerosol deposition magnitude bubbles, colored by aerosol type over Earth's land ice areas.  This example plot shows glacier thinning magnitude.  
![glacierthinning](https://cloud.githubusercontent.com/assets/18668993/16627725/8e515924-437c-11e6-86cc-cfbb96b92a52.jpg)

**Comment by karanjeets** [comment]
+1. I will review this and get back to you soon.
